gr-analog/examples/fm_demod.py

```diff
@@ -38,7 +38,7 @@ def __init__(self):
 
         # Set the demodulator using the same deviation as the receiver.
         max_dev = 75e3
-        fm_demod_gain = input_rate / (2 * math.pi * max_dev / 8.0)
+        fm_demod_gain = input_rate / (2 * math.pi * max_dev)
```

Comment:
Please clarify in a code comment: why the `/ 8.0`?

Reply:
This formula is not commented elsewhere : 
- ```k = 2 * math.pi * max_dev / quad_rate``` for instance in wfm_tx.py
- ```fm_demod_gain = quad_rate / (2 * math.pi * max_dev)``` in wfm_rcv.py

My understanding is that 8/ factor is not needed. The impact is a lower volume (scale factor) in the audio received.